lib/trento/activity_logging/parser/activity_parser.ex

```diff
@@ -54,4 +61,51 @@ defmodule Trento.ActivityLog.Parser.ActivityParser do
 
   defp parse_domain_event_activity_info(:metadata, activity, activity_context),
     do: EventParser.get_activity_metadata(activity, activity_context)
+
+  defp map_severity_level(activity_type, metadata) do
```

Comment:
As agreed offline we can move these functions in `Trento.ActivityLog.SeverityLevel` so that the severity detection module can be tested in isolation.